core/src/main/java/ai/timefold/solver/core/impl/domain/variable/declarative/EmptyVariableReferenceGraph.java

```diff
@@ -26,4 +26,9 @@ public String toString() {
         return "{}";
     }
 
+    @Override
+    public void setUnknownInconsistencyValues() {
```

Comment:
Let's try to keep boilerplate methods (such as `toString`) last.
Also, I wonder if this method should have been made `default` on the interface, avoiding the need for such overrides.